openapi/components/schemas/users/update-user-payload.json

```diff
@@ -0,0 +1,115 @@
+{
+  "UpdateUserRequestModel": {
+    "type": "object",
+    "properties": {
+      "phoneNumbers": {
+        "type": "array",
+        "items": {
+          "type": "string",
```

Comment:
This is also required in many places. Keep it in a common place.

Reply:
updated in 7510b59ea5622893a67d478c91caa3b141d55eea